cmake/dependencies.cmake

```diff
@@ -53,6 +53,12 @@ fetch_dep(rapidjson
   TAG 14a5dd756e9bef26f9b53d3b4eb1b73c6a1794d5
   SOURCE_SUBDIR redpanda_build)
 
+fetch_dep(unordered_dense
+  REPO https://github.com/redpanda-data/unordered_dense
+  TAG 9338f301522a965309ecec58ce61f54a52fb5c22
+  SOURCE_DIR redpanda_build
```

Comment:
this directory doesn't exist in the repo copy paste error?

Reply:
Also you need to add this to MakeAvailable

Reply:
Ah brilliant, yeah I was still experimenting with this (== guessing).